telegram/_chat.py

```diff
@@ -355,6 +359,86 @@ def de_json(cls, data: Optional[JSONDict], bot: "Bot") -> Optional["Chat"]:
 
         return super()._de_json(data=data, bot=bot, api_kwargs=api_kwargs)
 
+    def mention_markdown(self, name: str = None) -> str:
+        """
+        .. versionadded:: 20.0
+
+        Note:
+            :tg-const:`telegram.constants.ParseMode.MARKDOWN` is a legacy mode, retained by
+            Telegram for backward compatibility. You should use :meth:`mention_markdown_v2`
+            instead.
+
+        Args:
+            name (:obj:`str`): The name used as a link for the chat. Defaults to :attr:`full_name`.
+
+        Returns:
+            :obj:`str`: The inline mention for the chat as markdown (version 1).
+
+        """
+        if self.type == self.PRIVATE:
+            if name:
+                return helpers_mention_markdown(self.id, name)
+            if self.full_name:
+                return helpers_mention_markdown(self.id, self.full_name)
+            raise TypeError("Can not create a mention to a private chat without first name")
+        if self.username:
+            if name:
+                return f"[{name}]({self.link})"
+            if self.full_name:
+                return f"[{self.full_name}]({self.link})"
```

Comment:
Guess that's right, `full_name` is `None` for non-private chats. Instead of username (which also may be `None`), we should use `title`, I guess

Reply:
> Guess that's right, `full_name` is `None` for non-private chats. Instead of username (which also may be `None`), we should use `title`, I guess

Isn't `self.username` always `not None` if the chat is public chat?

Reply:
title is probably a better "name" than the username